anime_downloader/sites/animesvostfr.py

```diff
@@ -0,0 +1,247 @@
+            
+            results = []
+            
+            for v in soup.select('div.movies-list > div.ml-item'):
+                t = v.h2.text
+                u = v.a['href']
+                p = v.img['src']
+                
+                lang = v.select_one('div.language').text.upper()
+                
+                #logger.debug( "results: {}".format(lang) )
+                
+                eps = v.select_one('span.mli-quality').text.upper()
+                
+                #logger.debug( "results: {}".format(eps) )
+                
+                if re.search('VOSTFR', lang, re.IGNORECASE):
+                    t += ' (VOSTFR '+eps+')'
+                elif re.search('VF', lang, re.IGNORECASE):
+                    t += ' (VF '+eps+')'
+                elif re.search('VO', lang, re.IGNORECASE):
+                    t += ' (VO '+eps+')'
+                else:
+                    t += ' (V? '+eps+')'
+                    
+                
+                search_result_info = SearchResult(title=t, url=u, poster=p)
+               
+                results.append(search_result_info)
+
+            
+            logger.debug( "results: {}".format(results) )
+            
+            return results
+        
+        def _scrape_episodes(self):
+            soup = helpers.soupify(helpers.get(self.url))
+
+            episode_links = None
+            results = [] 
+            
+            season = soup.find("div", id="seasonss")
+            
+            if season:
+                episode_links = season.select('div.les-title > a')
+                
+            
+            if episode_links:
+            
+                logger.debug( "episode_links: {}".format(episode_links) )
+                
+                #results = [a.get('href') for a in episode_links[::-1]]
+                results = [a.get('href') for a in episode_links]
+                
+                logger.debug( "results: {}".format(results) )
+            else:
+                results.append(self.url)
+                
+            
+            return results
+
+        def _scrape_metadata(self):
+            
+            logger.debug("_scrape_metadata url: %s", str(self.url) )
+            
+            soup = helpers.soupify(helpers.get(self.url, cf=True))
+     
+            h1 = soup.select_one('div.main-content h1')
+            logger.debug("_scrape_metadata H1: %s", str(h1))
+            
+            self.title = h1.text.strip()
+            
+
+class AnimesVOSTFREpisode(AnimeEpisode, sitename='animesvostfr'):
+        episodeId_url = 'https://www1.animesvostfr.net/api/episode'
+        stream_url = 'https://www1.animesvostfr.net/api/videos?episode_id'
+        anime_url = 'https://www.www1.animesvostfr.net/shows'
+        
+        link_stream_url = "https://www1.animesvostfr.net/ajax-get-link-stream/"
+        
+        SERVERS = [
+            'rapidvideo',
+            'photoss'
```

Comment:
What are you using this for? A quick ctrl-f shows that you aren't actually using this

Reply:
yes for the moment that is not use because i don't understand how is choose the good extractor

Reply:
"good extractor" which refers to, selected by user? One with selected language? Selected quality? Best quality? No 404? Fastest?